app/components/dialogs/migrate-beta.js

```diff
@@ -66,9 +66,9 @@ export default Component.extend({
 });
 
 function makeOptionFromAccount(account) {
-  const { id, title, isMigrationBetaAccepted, isMigrationBetaRequested, isOrganization, isUser } = account;
+  const { id, title, isMigrationBetaAccepted, isOrganization, isUser } = account;
   const isNotAdmin = isOrganization && !account.get('permissions.admin');
-  const state = isMigrationBetaAccepted ? 'subscribed' : isMigrationBetaRequested ? 'waitlisted' : isNotAdmin ? 'not admin' : '';
-  const disabled = isOrganization && (isMigrationBetaAccepted || isMigrationBetaRequested) || isUser || isNotAdmin;
+  const state = isMigrationBetaAccepted ? 'subscribed' : isNotAdmin ? 'not admin' : '';
+  const disabled = isOrganization && (isMigrationBetaAccepted) || isUser || isNotAdmin;
```

Comment:
Just a tiny visual tweak 😉 
```suggestion
  const disabled = isOrganization && isMigrationBetaAccepted || isUser || isNotAdmin;
```